tracing/src/main/java/com/palantir/remoting3/tracing/Tracer.java

```diff
@@ -202,10 +204,13 @@ private static Span toSpan(OpenSpan openSpan, Map<String, String> metadata) {
      */
     public static synchronized SpanObserver subscribe(String name, SpanObserver observer) {
         if (observers.containsKey(name)) {
-            log.warn("Overwriting existing SpanObserver with name {} by new observer: {}", name, observer);
+            log.warn("Overwriting existing SpanObserver with name {} by new observer: {}",
+                    UnsafeArg.of("name", name),
+                    UnsafeArg.of("observer", observer));
```

Comment:
logging this doesn't really make sense since SpanObserver doesn't have a toString... we can leave as-is and wrapped in UnsafeArg given this PR isn't supposed to change functionality though